### ISSUE TITLE
Delete duplicates AND first occurance

### ISSUE THREAD

**Issue by geerew** [jianbingfang/vscode-dup-checker#2]
Hi,

Firstly, thank you for the extension. It's quiet useful to me.

Is is possible for you to add an additional command to 'check for duplicates and delete all (including first occurrence)'?

```text
1 | aaa
2 | bbb
3 | bbb
4 | bbb
5 | ccc
```

In the above example, lines 2, 3 and 4 would be deleted, instead of just lines 3 and 4.

I understand this might be outside of the scope of your extension.

**Comment by jianbingfang** [comment]
Hi @geerew , it makes sense to me, i'm considering to add another parameter `removeAllDuplicates` in configuration to achieve this:
```
"dupchecker": {
  "trimStart": true,      // trim starting whitespaces in each line, default: true
  "trimEnd": true,        // trim ending whitespaces in each line, default: true
  "ignoreCase": false,    // ignore case when comparing lines, default: false
  "leaveEmptyLine": true,  // leave an empty line after removing duplicates if true, or remove whole line(including line break) if false, default: true
  "removeAllDuplicates": false // remove all duplicate lines including the first occurrence if true, default: false
}
```

I'm not going to add a new command because I think it's improper to mention any action in the first stage (just for checking).

What I can guess more from your idea is that you want just one stage to get both checking and removing done right?

Please share your thought if I missed any.

**Comment by geerew** [comment]
Thanks @jianbingfang.

Config is a good way to do it. 

Correct. Running the 'Check Duplicates' with the removeAllDuplicates: true will remove everything (including first occurrence).

Thanks.

**Comment by jianbingfang** [comment]
Hi @geerew , I have published new version 0.0.9 with configuration `removeAllDuplicates`, you should be able to update now, please have a try.

And I'll close this issue if you have no further question : )

**Comment by geerew** [comment]
Working. 

Thank you
